nodes/src/nodes/anonymize/glinerRecognizer.py

```
@@ -234,9 +234,6 @@ def handleClassifications(self, classifications: dict, target_object_text: str,
         labels = self.ruleParser.get_rules_names(unique_id_refs) + rules
 
         # Extract existing matches from classifications (offset, length tuples)
-        existing_matches = list(
-            (m['offset'], m['length']) 
-            for m in ((m.get('location', {}).get('inChars') or m) for m in text_matches)
-        )
+        existing_matches = list((m['offset'], m['length']) for m in ((m.get('location', {}).get('inChars') or m) for m in text_matches))
```

Ruff (0.15.7)

[warning] Unnecessary generator (rewrite as a list comprehension)

Rewrite as a list comprehension

(C400)